cypress-tests/cypress/constants/selectors/common.js

```diff
@@ -6,6 +6,7 @@ export const commonSelectors = {
   toastMessage: ".go3958317564",
   oldToastMessage: ".go318386747",
   newToastMessage: '.drawer-container > [style="position: fixed; z-index: 9999; inset: 16px; pointer-events: none;"] > .go4109123758 > .go2072408551 > .go3958317564',
+  duplicateEmail: '[data-cy="modal-message"]',
```

Comment:
Use the existing selector `modalMessage`

Reply:
fixed